fdbmonitor/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 set(FDBMONITOR_SRCS ConvertUTF.h SimpleIni.h fdbmonitor.cpp)
 
 add_executable(fdbmonitor ${FDBMONITOR_SRCS})
+# FIXME: This include directory is an ugly hack. We probably want to fix this
+# as soon as we get rid of the old build system
```

Comment:
We should probably fix this before removing the old build system, not after. #1000 is to address this. Can you please add #1000 as blocker or subtask of #1035?

Reply:
Done.